swingtix/bookkeeper/account_api.py

```diff
@@ -139,25 +139,25 @@ def get_bookset(self):  # pragma: no coverage
     def _DEBIT_IN_DB(self):
         return 1
 
-    def debit(self, amount, credit_account, description, debit_memo="", credit_memo="", datetime=None):
+    def debit(self, amount, credit_account, description, debit_memo="", credit_memo="", *args, **kwargs):
```

Comment:
How about dropping the *args?  My reasoning is there are too many positional arguments as it is, so any additional arguments would be better served as named arguments.